### ISSUE TITLE
No way to customize ``view_name`` in html templace

### ISSUE THREAD

**Issue by bubenkoff** [tomchristie/flask-api#49]
Currently it's generated by simple
```python
def convert_to_title(name):
    return name.replace('-', ' ').replace('_', ' ').replace('.', ' ').capitalize()
```
but for example if endpoint contains `.` it becomes strange

**Comment by rainulf** [comment]
I don't think this needs customisation. My preference is to actually remove `view_name`, and always use endpoint relative path as header title for the browsable page (eg. `/user` will be the header title).
